packages/pancake-uikit/src/components/PancakeToggle/StyledPancakeToggle.tsx

```diff
@@ -22,6 +22,25 @@ const scaleKeyValues = {
     butterSmearTwoRight: "2.5px", // these values adjust the position of it
   },
   md: {
+    pancakeSize: "26px",
```

Comment:
FYI @hachiojidev - I've added a new scale for the Pancake toggle - what was previously 'md' is now 'lg', and 'md' has some new styles.

It's only used by the menu